vhdbuilder/release-notes/AKSUbuntu/gen1/1804/2021.07.31.txt

```diff
@@ -0,0 +1,965 @@
+uidmap/bionic-updates,now 1:4.5-1ubuntu2 amd64 [installed,automatic]
+unattended-upgrades/bionic-updates,now 1.1ubuntu1.18.04.14 all [installed,automatic]
+update-manager-core/bionic-updates,now 1:18.04.11.13 all [installed,automatic]
+update-notifier-common/bionic-updates,now 3.192.1.11 all [installed,automatic]
+ureadahead/bionic-updates,now 0.100.0-21 amd64 [installed,automatic]
+usbutils/bionic,now 1:007-4build1 amd64 [installed,automatic]
+util-linux/bionic-updates,bionic-security,now 2.31.1-0.4ubuntu3.7 amd64 [installed]
+uuid-runtime/bionic-updates,bionic-security,now 2.31.1-0.4ubuntu3.7 amd64 [installed,automatic]
+vim/bionic-updates,bionic-security,now 2:8.0.1453-1ubuntu1.4 amd64 [installed,automatic]
+vim-common/bionic-updates,bionic-security,now 2:8.0.1453-1ubuntu1.4 all [installed,automatic]
+vim-runtime/bionic-updates,bionic-security,now 2:8.0.1453-1ubuntu1.4 all [installed,automatic]
+vim-tiny/bionic-updates,bionic-security,now 2:8.0.1453-1ubuntu1.4 amd64 [installed,automatic]
+walinuxagent/bionic-updates,now 2.2.45-0ubuntu1~18.04.1 amd64 [installed]
+wget/bionic-updates,bionic-security,now 1.19.4-1ubuntu2.2 amd64 [installed,automatic]
+whiptail/bionic,now 0.52.20-1ubuntu1 amd64 [installed,automatic]
+wireless-regdb/bionic-updates,bionic-security,now 2020.11.20-0ubuntu1~18.04.1 all [installed]
+xauth/bionic,now 1:1.0.10-1 amd64 [installed,automatic]
+xdelta3/bionic,now 3.0.11-dfsg-1ubuntu1 amd64 [installed,automatic]
+xdg-user-dirs/bionic,now 0.17-1ubuntu1 amd64 [installed,automatic]
+xfsprogs/bionic,now 4.9.0+nmu1ubuntu2 amd64 [installed,automatic]
+xkb-data/bionic-updates,now 2.23.1-1ubuntu1.18.04.1 all [installed,automatic]
+xxd/bionic-updates,bionic-security,now 2:8.0.1453-1ubuntu1.4 amd64 [installed,automatic]
+xz-utils/bionic,now 5.2.2-1.3 amd64 [installed]
+zerofree/bionic,now 1.0.4-1 amd64 [installed,automatic]
+zip/bionic,now 3.0-11build1 amd64 [installed]
+zlib1g/bionic,now 1:1.2.11.dfsg-0ubuntu2 amd64 [installed,automatic]
+=== Installed Packages End
+Disk usage:
+Filesystem      Size  Used Avail Use% Mounted on
+udev            3.4G     0  3.4G   0% /dev
+tmpfs           696M  708K  695M   1% /run
+/dev/sdb1        29G   22G  7.6G  75% /
+tmpfs           3.4G     0  3.4G   0% /dev/shm
+tmpfs           5.0M     0  5.0M   0% /run/lock
+tmpfs           3.4G     0  3.4G   0% /sys/fs/cgroup
+/dev/sdb15      105M  6.6M   98M   7% /boot/efi
+/dev/sda1        14G   41M   13G   1% /mnt
+tmpfs           696M     0  696M   0% /run/user/1000
+Using kernel:
+Linux version 5.4.0-1055-azure (buildd@lcy01-amd64-025) (gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)) #57~18.04.1-Ubuntu SMP Fri Jul 16 19:40:19 UTC 2021
+Install completed successfully on  Sat Jul 31 03:36:05 UTC 2021
+VSTS Build NUMBER: 20210731.1_master_45141220
+VSTS Build ID: 45141220
+Commit: e63a8117a096aba4ce58a90b5ef287d64141321d
+Ubuntu version: 18.04
+Hyperv generation: V1
+Feature flags: None
+Container runtime: docker
+FIPS enabled: 
```

Comment:
we should make sure these values are populated consistently...some have FIPS enabled: false, some have empty string (both disabled), some have Ubuntu version empty, some don't...